BigQuery/tests/System/ManageTablesTest.php

```diff
@@ -213,6 +228,28 @@ public function testCopiesExternalTable(Table $externalTable)
         $this->runJob($copyJobConfig);
     }
 
+    public function testIam()
+    {
+        $table = self::createTable(self::$dataset, uniqid(self::TESTING_PREFIX));
+        $iam = $table->iam();
+
+        $policy = [
+            'bindings' => [
+                [
+                    'role' => 'roles/bigquery.admin',
+                    'members' => [
+                        'user:gcloud.php.tests@gmail.com'
+                    ]
+                ]
+            ]
+        ];
+        $iam->setPolicy($policy);
+        $actualPolicy = $iam->reload();
+
+        $this->assertEquals($actualPolicy, $iam->policy());
+        $this->assertEquals($policy['bindings'][0], $actualPolicy['bindings'][0]);
```

Comment:
Consider exercising the testIamPermissions as well?  Though I recall mentioned PHP testing uses two identities, so it may be more complex to assert permissions if the probing identity has less access.

Reply:
Done. Thanks for flagging that.